ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -341,6 +390,21 @@ public ThreeDSecureV1UiCustomization getV1UiCustomization() {
         return v1UiCustomization;
     }
 
+    /**
+     * @return The UI type.
+     */
+    @Nullable
```

Comment:
```suggestion
```
Shouldn't need since `int` by itself is non-nullable.

Reply:
Updated: https://github.com/braintree/braintree_android/pull/738/commits/b1313fc2f152519e3fea12c0de3b51e48e1364f7